exercises/concept/the-farm/.docs/hints.md

```diff
@@ -0,0 +1,35 @@
+# Hints
+
+## 1. Divide the food evenly
+
+- Start by writing the function signature of `DivideFood`.
+  It should accept 2 parameters of type `FodderCalculator` and `int` and return two values of types `float64` and `error`.
+  Revisit the [functions concept][concept-functions] if you need more information on how to define functions.
+- In the function body, call the `FodderAmount` [method][concept-methods] on `FodderCalculator` to fetch the default total amount of fodder for the cows.
+  It will return the actual result and an error.
+  Handle the error via an if-statement as it was explained in the introduction.
+- After that, call the `FatteningFactor` method and handle the error return value as before.
+- Now that you have the fodder amount and the factor, you can calculate the final result.
+  You need to divide the fodder by the number of cows (revisit [numbers] for hints on type conversion) and multiply with the factor. Check the introduction for what to return as the error value in case of success.
+
+## 2. Check the number of cows
+
+- `ValidateInputAndDivideFood` has the same function signature as `DivideFood`.
+- Since you want to return early in case of an error in Go, you first check whether the number of cows is less or equal than 0 with an if-statement.
```

Comment:
```suggestion
- Since you want to return early in case of an error, you first check whether the number of cows is less or equal than 0 with an if-statement.
```